### PR TITLE
Add liveness check for workers

### LINKED ISSUE #5885: Redash Workers Becomes Inactive Automatically
### Issue Summary

Everyday we see `redash` workers become inactive/stale and they are unable to process queries.
This has mostly happened for adhoc workers and sometimes for other workers too.
We have to restart the workers everyday and then everything works fine again. This is very big pain.

### Steps to Reproduce

No clear steps are there to reproduce 
But one scenario could be if when workers are about to start and connect to redis instance if they are unable to connect then worker health check fails and they never come into alive state.

This is a bug because it requires manual restart always.

### Technical details:

* Redash Version: redash/redash:10.1.0.b50633 
* Browser/OS: GKE
* How did you install Redash: Helm Charts

### Further Investigation
This has been reported earlier as well with different title and problems
https://github.com/getredash/contrib-helm-chart/issues/124
https://github.com/getredash/contrib-helm-chart/issues/72
https://github.com/getredash/redash/issues/5801

We should add a liveness probe to make restarts automatic when supervisord workers are dead.

### PR DESCRIPTION
**Pull request by shubhwip**
closes #5885

liveness check is based on
rq info command returns monitoring status of redash workers and other status checks
Sample Output
```
redash@redash-adhocworker-778fc5dbb6-jpscq:/app$ rq info --url $REDASH_REDIS_URL -R
queries      | 0
default      | 2
periodic     |██████████████████ 748
emails       | 0
schemas      | 0
5 queues, 750 jobs total

queries:  1a6d953d53984424ab775d3647427e7b (idle), 29dbf1ec1f774bacbd91ca36ccef6e43 (idle)
default:  –
periodic: –
emails:   –
schemas:  –
2 workers, 5 queues
```

## What type of PR is this? 

- [ ] Refactor
- [x] Feature
- [x] Bug Fix
- [ ] New Query Runner (Data Source) 
- [ ] New Alert Destination
- [ ] Other

## How is this tested?

- [ ] Unit tests (pytest, jest)
- [ ] E2E Tests (Cypress)
- [x] Manually
- [ ] N/A

I have run rq info command in k8s pods and based on that test i have written a healthcheck.

## Related Tickets & Documents

Similar Reported Issues Reference :
https://github.com/getredash/contrib-helm-chart/issues/124
https://github.com/getredash/contrib-helm-chart/issues/72
https://github.com/getredash/redash/issues/5801

## Additional Comments
After this is merged, next change is in helm charts repo 
https://github.com/getredash/contrib-helm-chart/pull/140

### CODE REVIEW

**Comment by lucasfcnunes** [comment]
@shubhwip Can you trigger the CI to test if there still are failings?

**Comment by shubhwip** [comment]
@lucasfcnunes I triggered it again but not sure why tests are failing, when I didn't make a change in tests at all.

**Comment by shubhwip** [comment]
@lucasfcnunes Can I draw your attention on this as this issue is faced by many people and myself everyday ?

File: bin/worker_healthcheck.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+# 5 queues, 750 jobs total
+
+# queries:  1a6d953d53984424ab775d3647427e7b (idle), 29dbf1ec1f774bacbd91ca36ccef6e43 (idle)
+# default:  –
+# periodic: –
+# emails:   –
+# schemas:  –
+# 2 workers, 5 queues
+
+WORKERS_COUNT=${WORKERS_COUNT}
```
Comment (arikfr) [inline bin/worker_healthcheck.sh:18]: Thank you for adding this! Is there a reason we need the separate script?

Comment (shubhwip) [reply bin/worker_healthcheck.sh:18]: Yes we don't need it. Earlier I thought we would use this separate script in helm chart's liveness check.
But I realised there used to be healthcheck in docker entrypoint script.
I will remove this script.

**Review by arikfr — APPROVED** [review]

**Pull request merged**